app/streaming/input.cpp

```diff
@@ -775,9 +775,20 @@ void SdlInputHandler::handleControllerDeviceEvent(SDL_ControllerDeviceEvent* eve
         state->jsId = SDL_JoystickInstanceID(SDL_GameControllerGetJoystick(state->controller));
         state->haptic = SDL_HapticOpenFromJoystick(SDL_GameControllerGetJoystick(state->controller));
         state->hapticEffectId = -1;
-        if (state->haptic != nullptr && (SDL_HapticQuery(state->haptic) & SDL_HAPTIC_LEFTRIGHT) == 0) {
-            SDL_HapticClose(state->haptic);
-            state->haptic = nullptr;
+        state->hapticMethod = GAMEPAD_HAPTIC_METHOD_NONE;
+        if (state->haptic != nullptr) {
+            if ((SDL_HapticQuery(state->haptic) & SDL_HAPTIC_LEFTRIGHT) == 0) {
+                if (SDL_HapticRumbleSupported(state->haptic)) {
+                    SDL_HapticRumbleInit(state->haptic);
+                    state->hapticMethod = GAMEPAD_HAPTIC_METHOD_SIMPLERUMBLE;
+                    SDL_HapticRumblePlay(state->haptic, 0.75, 350);
```

Comment:
Looks like `SDL_HapticRumblePlay()` was left here by mistake

Reply:
I thought a small effect could be useful to understand which method was involved.

Reply:
I'd prefer a log message instead. It might get annoying for some users to have their gamepads rumble on their own every time they start a connected (especially if they have unused gamepads attached which may be resting on a table or other surface that could be loud when it rumbles).

Reply:
Fixed